src/loader/parse-migrations.ts

```diff
@@ -31,7 +31,10 @@ function validateFileName(name: string): void {
 
 function isStatementSupportedByEmulator(statement: string): boolean {
   const notSupportedPatterns: ((statement: string) => boolean)[] = [
-    (statement) => /^alter table [a-z][\d\w_]{0,128} add row deletion policy .+$/i.test(statement),
+    (statement) =>
+      /^(alter table [a-z][\d\w_]{0,128} add row deletion policy .+$|create view|create or replace view|drop view|create role|drop role|grant|revoke)/i.test(
```

Comment:
Tested locally with the emulator to check all the VIew DDL statements, as well as the newly added role-related ones which were also unsupported in Emulator and errored.